sources/packages/backend/apps/workers/test/jest-e2e.json

```diff
@@ -14,7 +14,6 @@
   "setupFiles": [
     "<rootDir>/../../env-setup.js"
   ],
-  "collectCoverage": true,
```

Comment:
Why did we remove the collectCoverage from all jest-e2e files?

Reply:
It was not removed it was actually moved.
How it was configured, it generated the coverage every time even while working on the E2E test, debugging, and all, which is pretty annoying.
The parameter is now part of the npm script (e.g. `test:e2e:api`).
```json
jest --collect-coverage --config ./apps/api/test/jest-e2e.json --forceExit
```